common/constants.js

```diff
@@ -212,6 +214,12 @@ module.exports = Object.freeze({
   ANNOTATION_KEYS: {
     STATE: 'state',
     OPTIONS: 'options',
+    RESULT: 'result'
+  },
+  APISERVER: {
+    STATE: {
+      IN_QUEUE: 'in_queue'
+    },
```

Comment:
Include all possible states. 
keep note of spaces (spaces are not allowed in apiserver labels)